ddl/ddl_api.go

```diff
@@ -106,6 +106,13 @@ func checkTooLongTable(table model.CIStr) error {
 	return nil
 }
 
+func checkTooLongIndex(index model.CIStr) error {
+	if len(index.L) > mysql.MaxIndexIdentifierLen {
+		return ErrTooLongIdent.Gen("too long index %s", index)
```

Comment:
Is there a suitable error message in errname.go?

Reply:
ok. I changed.